src/foam/nanos/auth/RetrievePassword.js

```diff
@@ -93,7 +94,8 @@ foam.CLASS({
         const user = this.User.create({ email: this.email });
         this.resetPasswordToken.generateToken(null, user).then((_) => {
           this.ctrl.add(this.NotificationMessage.create({
-            message: `${this.INSTRUC_ONE} ${this.email}. ${this.INSTRUC_TWO}`,
+            message: `${this.INSTRUC_ONE_TITLE}`,
```

Comment:
I think this is ridiculous in terms of redundancy, but if you @sarthak-marwaha think this is appropriate UI/UX, then fine.

Reply:
It is still two messages just split up into two rows now. I believe we can delete the 'INSTRUC_ONE' if it not required